input/pagecontent/index-dak.md

```diff
@@ -0,0 +1,56 @@
+<!---Note: Remove this below div element for releasing, only to maintain for main branch -->
+<div>
+<p> This DAK and set of implementation tools are still undergoing development. </p>
+<p> Content is for demonstration purposes only. </p>
+</div>{:.stu-note}
+
+
+### Summary 
+This WHO <mark>[insert health domain here]</mark> DAK <mark>add content here</mark>
+
+### L1 Narrative Guidelines
+<mark>[insert content here]</mark>
+
+### L2 DAK 
+The L2 <mark>[insert health domain here]</mark> DAK publications and implementations tools can be found here:
+
+<!-- insert links inside the parentheses below --->
+- [Published DAK Document]()
+
+- [Link to core data dictionary]()
+ 
+- [Link to decision support logic]() 
+
+- [Link to scheduling logic]() 
+
+- [Link to indicators table]()
+ 
+- [Link to functional and non functional requirements]()
+
+### L3 FHIR Implementation Guide
+The L3 FHIR Implementation Guide for the <mark>[insert health domain here]</mark> SMART Guidelines is yet to be published. Links will be published here as soon as they're available.
+
+### L4 Reference Implementations
+Reference implementations representing the L4 layer for the <mark>[insert health domain here]</mark> SMART Guidelines are not yet available. Links will be published here as soon as they're available.
+
```

Comment:
Also please add: 

L5 Dynamic guidelines 
Content representing the L5 layer for the <mark>[insert health domain here]</mark> SMART Guidelines are not yet available. Links will be published here as soon as they're available.